models/v1/gardens/gardens_models.go

```diff
@@ -0,0 +1,8 @@
+package gardens_models
+
+type Rules struct {
+	ID             string `bson:"_id" json:"_id"`
```

Comment:
@yukari-n-erb You can change it in another PR but I found out you can use the type primitive.ObjectID so go will know it's an ObjectID. You'll also need to include `,omitempty` after `bson:"_id"` or else the create garden will be a bit strange (I just went through this myself :D).
So it should look like this `bson:"_id,omitempty" json:"_id"`